src/schema.graphql

```diff
@@ -1057,3 +1057,9 @@ type DistributorClaimedAmount @entity {
   "The total amount of the token claimed by the user"
   cumulativeClaimedAmount: BigInt!
 }
+
+"The distributor claimed amounts"
+type NetworkValidator @entity {
```

Comment:
I suggest to make it immutable and use Bytes as ID. See https://thegraph.com/docs/en/subgraphs/cookbook/immutable-entities-bytes-as-ids/